test.py

```diff
@@ -195,6 +195,9 @@ def parse_cmd_line() -> argparse.Namespace:
                         help='Let me manually run the test executable at the moment this script would run it')
     parser.add_argument('--byte-limit', action="store", default=None, type=int,
                         help="Specific byte limit for failure injection (random by default)")
+    parser.add_argument('--run-internet-dependent-tests',
+                        dest="run_internet_dependent_tests", action="store_true", default=False,
+                        help="Run tests which need to download something from the internet.")
```

Comment:
Tests should default to True. Most people do have internet.

The reason I dislike Internet dependency is not because I don't have an internet connection, it's because the tests can break when artifacts are changed or removed.

Reply:
Done.